api/external/exports.html

```diff
@@ -265,6 +265,12 @@ <h1>Exports<a class="headerlink" href="#exports" title="Permalink to this headli
         <li><strong>sms</strong> (<em>dictionary</em>) &#8211; Describes what should be exported from SMS related fields. </li>
```

Comment:
Oops, I totally missed this in the other PR, but to keep this valid HTML the `<li>` should contain the `<ul>`:


```suggestion
        <li><strong>sms</strong> (<em>dictionary</em>) &#8211; Describes what should be exported from SMS related fields.
```